BDArmory/Control/MissileFire.cs

```diff
@@ -3529,11 +3553,11 @@ bool CheckBombClearance(MissileBase ml)
                     float radius = launcher.decoupleForward ? launcher.ClearanceRadius : launcher.ClearanceLength;
                     float time = Mathf.Min(ml.dropTime, 2f);
                     Vector3 direction = ((launcher.decoupleForward
-                        ? ml.MissileReferenceTransform.transform.forward
-                        : -ml.MissileReferenceTransform.transform.up) * launcher.decoupleSpeed * time) +
+                        ? referenceTransform.transform.forward
```

Comment:
Isn't `referenceTransform.transform` redundant? I.e., `referenceTransform.transform.up` should be the same as just `referenceTransform.up` shouldn't it?

Reply:
So it is. Fixed.